packages/gatsby/cache-dir/loader.js

```diff
@@ -270,28 +271,37 @@ export class BaseLoader {
 
       const finalResult = {}
 
-      const componentChunkPromise = this.loadComponent(componentChunkName).then(
-        component => {
-          finalResult.createdAt = new Date()
-          let pageResources
-          if (!component) {
-            finalResult.status = PageResourceStatus.Error
-          } else {
-            finalResult.status = PageResourceStatus.Success
-            if (result.notFound === true) {
-              finalResult.notFound = true
+      let componentChunkPromise
+      if (
+        process.env.NODE_ENV === `development` &&
+        process.env.GATSBY_EXPERIMENTAL_LAZY_DEVJS &&
+        options.isPrefetch === true
+      ) {
+        componentChunkPromise = Promise.resolve()
```

Comment:
Why do we resolve here? Is it because we don't want to build js for prefetches? If so - it is not immediately obvious - would be great to add a comment here.

Reply:
yes & adding comment